000458/FlatironInstitute/PSTH_neurosift_exploration.md

```diff
@@ -0,0 +1,39 @@
+# 000458 - PSTH Neurosift Exploration
+
+See [001_summarize_contents.ipynb](./001_summarize_contents.ipynb) for a summary of the contents of the NWB files in this Dandiset, including the trial information. You can use that to decide which NWB files to examine. Here's an example of one that targets the motor cortex:
+
+```
+Asset 7: sub-551397/sub-551397_ses-20210211_behavior+ecephys.nwb -- size: 15202.44 MB
+in vivo electrophysiology in a head-fixed mouse during cortical electrical microstimulation
+EEG and Neuropixels recording during wakefulness and isoflurane anesthesia
+single pulse electrical stimuli targeted to MOs (deep layers)
+  1800 trials
+  BEHAVIORAL EPOCHS
+    awake: 386 valid and not running
+    isoflurane: 480 valid and not running
+    recovery: 553 valid and not running
+  STIMULUS TYPES
+    electrical: 1103 valid and not running
+    visual: 316 valid and not running
+  STIMULUS DESCRIPTIONS
+    biphasic: 1103 valid and not running
+    white circle: 316 valid and not running
+  ESTIM TARGET REGIONS
+    MOs: 1103 valid and not running
+    n/a: 316 valid and not running
+```
+
+First navigate to this Dandiset on DANDI Archive: [000458](https://dandiarchive.org/dandiset/000458). On the right panel, click on "Files" and then find [subject 551397](https://dandiarchive.org/dandiset/000458/0.230317.0039/files?location=sub-551397&page=1). For that NWB file, click on "OPEN WITH -> Neurosift". That should bring you [here](https://neurosift.app/?p=/nwb&url=https://api.dandiarchive.org/api/assets/d966b247-2bac-4ef0-8b80-aae010f50c98/download/&dandisetId=000458&dandisetVersion=0.230317.0039).
+
+* In Neurosift, open the "Intervals" panel and then click "PSTH" for the "trials" table.
+* Select "Group trials by: behavioral_epoch" and "Sort units by: location".
+* For trials table, enter `is_valid === 1 && is_running === 0 && stimulus_type === "electrical"`. This will restrict the trials to those that are valid, not running, and have an electrical stimulus.
```

Comment:
```suggestion
* Select "Trials filter" in the bottom right and enter `is_valid === 1 && is_running === 0 && stimulus_type === "electrical"`. This will restrict the trials to those that are valid, not running, and have an electrical stimulus.
```